src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperAttribute.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Internal.Web.Utils;
+
+namespace Microsoft.AspNet.Razor.Runtime.TagHelpers
+{
+    public class TagHelperAttribute : IReadOnlyTagHelperAttribute
+    {
+        public TagHelperAttribute()
+        {
+        }
+
+        public TagHelperAttribute(string name, object value)
+        {
+            Name = name;
+            Value = value;
+        }
+
+        public string Name { get; set; }
+
+        public object Value { get; set; }
+
+        public static implicit operator TagHelperAttribute(string value)
```

Comment:
If you want that code sample to work, that's just another overload of the indexer on `Attributes` right? Implicit conversions can be really surprising.